l10n_eu_product_adr/models/adr_class.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Stefan Rijnhart <stefan@opener.amsterdam>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+from odoo.osv.expression import AND
+
+
+class AdrClass(models.Model):
+    _name = "adr.class"
```

Comment:
you can add a migration here as we renamed clasess

Reply:
Adding *only* the migration of this class would not be wrong, but not very useful by itself I'm afraid.